src/util/date-helper.ts

```diff
@@ -80,4 +80,8 @@ export namespace DateHelper {
         const endTime = dayjs(end, format);
         return endTime.diff(startTime, "minute");
     };
+
+    export const toDayjs = (date: string): Dayjs => {
```

Comment:
to circumvent the behaviour where `dayjs("")` results in invalid day instead of returning the current day like `dayjs(undefined)`